addons/sourcemod/translations/vip_core.phrases.txt

```diff
@@ -2,30 +2,27 @@
 {
 	"VIP_CHAT_PREFIX"
 	{
-		"en"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"fi"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"de"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"pt"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"pt_p"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"ru"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
-		"ua"		"{DEFAULT}[ {GREEN}VIP {DEFAULT}]"
+		"en"		"[VIP]"
+		"fi"		"[VIP]"
+		"de"		"[VIP]"
+		"pt_p"		"[VIP]"
+		"ru"		"[VIP]"
+		"ua"		"[VIP]"
 	}
 	"VIP_MENU_TITLE"
 	{
-		"en"		"| VIP MENU |"
-		"fi"		"| VIP VALIKKO |"
-		"de"		"| VIP MENÜ |"
-		"pt"		"| MENU VIP |"
-		"pt_p"		"| MENU VIP |"
-		"ru"		"| VIP МЕНЮ |"
-		"ua"		"| VIP МЕНЮ |"
+		"en"		"|VIP MENU|"
+		"fi"		"|VIP VALIKKO|"
+		"de"		"|VIP MENÜ|"
+		"pt_p"		"|MENU VIP|"
```

Comment:
ptyou can not delete "pt", there must be both "pt" and "pt_p"

Reply:
I know, I have seen that yesterday already, I had attached the new file at the beginning of the post the file, there is everything correct (i hope)
[vip_core.phrases.txt](https://github.com/R1KO/VIP-Core/files/6291498/vip_core.phrases.txt)

Reply:
commit this file to pull request